agent/scripts/newrelic.ini.template

```diff
@@ -1166,30 +1166,30 @@ newrelic.daemon.logfile = "/var/log/newrelic/newrelic-daemon.log"
 ;
 ;newrelic.framework.wordpress.hooks = true
 
-; Setting: newrelic.framework.wordpress.hooks_threshold
-; Type   : time specification string ("500ms", "1s750ms" etc)
+; Setting: newrelic.framework.wordpress.hooks.options
+; Type   : string (all_callbacks, plugin_callbacks, threshold)
 ; Scope  : per-directory
-; Default: 1ms
-; Info   : Sets the threshold above which the New Relic agent will record a
-;          wordpress hooks.
+; Default: all_callbacks
+; Info   : Sets the options how WordPress hooks are instrumented.
 ;
-;newrelic.framework.wordpress.hooks_threshold = 1ms
-
-; Setting: newrelic.framework.wordpress.plugins
-; Type   : boolean
-; Scope  : per-directory
-; Default: true
-; Info   : Indicates if WordPress plugins are to be instrumented.
+;          New Relic agent can provide different levels of insights into WordPress hooks.
+;          By default, all hook callbacks functions are instrumented ("all_callbacks").
+;          To reduce agent's overhead it is possible to limit the instrumentation to only
+;          plugin/theme callbacks ("plugin_callbacks"). Third option is to monitor hooks
+;          without instrumenting callbacks ("threshold"). This option does not give insights
+;          about plugins/themes.
 ;
-;newrelic.framework.wordpress.plugins = true
+;newrelic.framework.wordpress.hooks.options = all_callbacks
```

Comment:
INI string settings are typically surrounded in quotation marks
```suggestion
;newrelic.framework.wordpress.hooks.options = "all_callbacks"
```

Reply:
Resolved with 89bd36a76874a8b91bc9d677467bfd567f16c39f.